base/errorshow.jl

```diff
@@ -319,12 +319,15 @@ function showerror(io::IO, ex::MethodError)
                       "\nYou can convert to a column vector with the vec() function.")
         end
     end
+    Experimental.register_error_hint(sym_hint_handler, MethodError)
     Experimental.show_error_hints(io, ex, arg_types_param, kwargs)
     try
         show_method_candidates(io, ex, kwargs)
     catch ex
         @error "Error showing method candidates, aborted" exception=ex,catch_backtrace()
     end
+    handler = Experimental._hint_handlers[MethodError]
+    !isempty(handler) && pop!(handler)
```

Comment:
I don't think we need this. It should be fine to show the same message if a user makes the same mistake twice. This will also delete the wrong handler in case packages add extra hints here.